terraform/terraform.nuspec

```diff
@@ -22,6 +22,34 @@ The key features of Terraform are:
 For more information, see the [introduction section](http://www.terraform.io/intro) of the Terraform website.
     </description>
     <releaseNotes>
+## 0.10.2 (August 16, 2017)
+
+BUG FIXES:
+
+* tools/terraform-bundle: Add missing Ui to ProviderInstaller (fix crash) (#15826)
+* go-plugin: crash when server emits non-key-value JSON (go-plugin#43)
+
+##0.10.1 (August 15, 2017)
```

Comment:
I tend to only put the latest changes in the release notes. Towards the bottom of this tag there is a link to the full Changelog for that tag. Could you update that and remove the old release notes for < 0.10.2 ?